src/components/Connection.tsx

```diff
@@ -87,7 +96,15 @@ const Connection: React.FC<ConnectionProps> = ({
     const value = e.target.value.replace(/[^0-9]/g, "");
     setCustomTime(value);
   };
+  //Function to delete all saved files
+  const deletedata=()=>{
+    setDatasets([]);
+  };
 
+const deleteindividualfiles=(index: number)=>{
+  const newDatasets = datasets.filter((item, i) => i !== index);
+  setDatasets(newDatasets);
+}
```

Comment:
This functions just removes the element, it's ok to remove element but this will create an issue. If I as user did 6 recordings and i know that 2nd and 5th has to be deleted, after deleting, the file names will be file1,2,3,4 and not file1,3,4,6. We need the 2nd configuration, for ease of user to understand which data has been deleted and which is kept.